projects/storefrontapp/src/app/spartacus/features/bundle-feature.module.ts

```diff
@@ -0,0 +1,20 @@
+import { NgModule } from '@angular/core';
+import { provideConfig } from '@spartacus/core';
+import { BundleRootModule } from 'projects/core/src/cart/bundle/root/public_api';
+
+@NgModule({
+  imports: [BundleRootModule],
+  providers: [
+    provideConfig({
+      featureModules: {
+        bundle: {
+          module: () =>
+            import('projects/core/src/cart/bundle/public_api').then(
+              (m) => m.BundleModule
+            ),
+        },
+      },
+    }),
```

Comment:
The lazy loading of the feature is _triggered_ by the existence of the CMS components on the page. 
Let's not define the configuration for lazy loading of the feature until we have the working CMS components implemented.